lib/parsers/stream-json-parser.ts

```
@@ -0,0 +1,230 @@
+    // Clear any pending timeouts
+    this.resetCompletionTimeout();
+    callback();
+  }
+
+  override _destroy(error: Error | null, callback: (error?: Error | null) => void) {
+    // Clean up timeouts when stream is destroyed
+    this.resetCompletionTimeout();
+    callback(error);
+  }
+
+  private processBuffer(isFlush: boolean = false) {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)